info.go

```diff
@@ -127,6 +130,28 @@ func (pi *ProgramInfo) ID() (ProgramID, bool) {
 	return pi.id, pi.id > 0
 }
 
+// ProgramStats holds statistics of a program.
+type ProgramStats struct {
+	// Total accumulated runtime of the program ins ns.
+	RuntimeNS uint64
```

Comment:
```suggestion
	Runtime time.Duration
```

This is probably nicer?